src/ages/distributions/ageOneDistribution.ts

```diff
@@ -0,0 +1,17 @@
+import { computeMarketsEmissions, getGraphMarkets } from "../../utils";
+import { EpochConfig } from "../ages.types";
+
+export const ageOneDistribution = async ({
+  snapshotBlock,
+  totalEmission,
+  finalTimestamp,
+  initialTimestamp,
+  id,
+}: EpochConfig) => {
+  if (!snapshotBlock) throw Error(`Cannot compute distribution for epoch ${id}: snapshotBlock is missing`);
+  const duration = finalTimestamp.sub(initialTimestamp);
+  const ageOneMarketsParameters = await getGraphMarkets(snapshotBlock);
```

Comment:
The problem is that the need for a future age is not predictable

Reply:
No but I could think about sth like a list of ages configuration and a `currentAge` variable. whenever we wanna change age and update configuration, we only have to add the configuration to the list and increase the `currentAge`. Because I feel like whenever we wanna change age we have to go across the whole repo and replace `ageOne` by `ageTwo` which can lead to bugs if we forget to change one thing :/